init-docker-mssql-server.sh

```diff
@@ -0,0 +1,7 @@
+
+# try to create test database until 
+for  i in $(seq 30); do
+  echo 'Waiting for SQL Server to wake up';
+  docker-compose exec mssql /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P S0meVeryHardPassword -d master -Q 'CREATE DATABASE mssql_test_db' 2> /dev/null && break;
```

Comment:
This should be slightly faster as it only sleeps for 1s:

```bash
set +e
for i in {1..30}; do
	docker-compose exec -T mssql /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P 'S0meVeryHardPassword' -b -Q 'CREATE DATABASE mssql_test_db' -e
	if [ "$?" -ne 0 ]; then
		sleep 1s
	else
		DB_READY=1
		echo "mssql_test_db ready after $i attempts"
		break
	fi
done
set -e
```

Just copied this from an existing project - feel free to readapt for output redirection or keep the eccentric `-e / +e` ops.